voltixApp/VoltixApp/Views/Send/SendInputDetailsView.swift

```diff
@@ -209,7 +210,8 @@ struct SendInputDetailsView: View {
                                                         if tx.coin.ticker.uppercased() == "ETH" {
                                                             tx.amount = eth.addressInfo?.ETH.getAmountInEth(newValueDouble) ?? ""
                                                         } else {
-                                                            if let tokenInfo = eth.addressInfo?.tokens?.first(where: {$0.tokenInfo.symbol == "USDC"}) {
+                                                            if let tokenInfo = tx.token {
```

Comment:
This can be rewritten as:
else if let tokenInfo = tx.token {
// Code here
}

Reply:
Good idea.